TelCo.ColorCoder/ColorFromPairNumber.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace TelCo.ColorCoder
+{
+    class ColorFromPairNumber : Program

```

Comment:
Why derive from `Program`?

Reply:
So that i can use the color array defined inside the Program Class and don't have to define it again somewhere.